src/app/config/app-config.service.spec.ts

```diff
@@ -27,6 +27,7 @@ describe('AppConfigService', () => {
       await appConfigService.loadConfig()
       expect(httpClient.get).toHaveBeenCalledTimes(1)
       expect(appConfigService.config).toEqual(config)
+      expect(appConfigService.getConfig()).toEqual(config)
```

Comment:
Why the additional verification/comparison of config?

Reply:
I must implement the method getConfig() in order to mock its behavoiur (see feature.service.spec.ts) and actually, I want the config to be accessed only via getConfig().
However, in some cases it is addressed directly inside the Vitagroup code which for compatibility I do not want to change. Thus I must test both - the property 'config' and its accessor method 'getConfig()'